Author: Alastair McKinstry <mckinstry@debian.org>
Description: Workaround for compiler bug in _NGResolvePath() on Debian.
 Root cause still under examination
Last-Updated: 2014-07-26
Forwarded: not-needed

Index: ncl-6.2.0/ni/src/ncl/NclDriver.c
===================================================================
--- ncl-6.2.0.orig/ni/src/ncl/NclDriver.c
+++ ncl-6.2.0/ni/src/ncl/NclDriver.c
@@ -11,10 +11,10 @@ int quark_comp(const void *q1, const voi
 	return(strcmp((const char*)NrmQuarkToString(*(NrmQuark *)q1),(const char*)NrmQuarkToString(*(NrmQuark *) q2)));
 }
 
-char *preload_scripts[4] = {"$NCARG_ROOT/lib/ncarg/nclscripts/csm/gsn_code.ncl",
-                            "$NCARG_ROOT/lib/ncarg/nclscripts/csm/gsn_csm.ncl",
-                            "$NCARG_ROOT/lib/ncarg/nclscripts/utilities.ncl",
-                            "$NCARG_ROOT/lib/ncarg/nclscripts/csm/contributed.ncl"};
+char *preload_scripts[4] = {"/usr/lib/ncarg/nclscripts/csm/gsn_code.ncl",
+                            "/usr/lib/ncarg/nclscripts/csm/gsn_csm.ncl",
+                            "/usr/lib/ncarg/nclscripts/utilities.ncl",
+                            "/usr/lib/ncarg/nclscripts/csm/contributed.ncl"};
 
 int NclDriver(int argc, char **argv)
 {
